prompting-client/tests/integration.rs

```diff
@@ -92,6 +92,9 @@ macro_rules! expect_single_prompt {
             let id = pending.remove(0);
```

Comment:
shouldn’t we have at least one integration test exercising the camera interface path?